colour/models/rgb/transfer_functions/__init__.py

```diff
@@ -777,9 +777,9 @@ def encoding_cctf(value, function='sRGB', **kwargs):
 
     if 'itu-r bt.2100' in function.lower():
         usage_warning(
-            'With the "ITU-R BT.2100" method, only the inverse electro-optical'
+            'With the ITU-R BT.2100 method, only the inverse electro-optical '
```

Comment:
On adding a space, some checks failed with the following message:

```
E501 line too long (80 > 79 characters)
            'With the "ITU-R BT.2100" method, only the inverse electro-optical '
                                                                               ^
```

To fix this, I removed the double quotes around ITU-R BT.2100 at both places.

Reply:
We should keep them ideally for emphasis, is it possible to actually wrap the lines please?

Cheers,

Reply:
Sure! It's done. Do the current changes work?

Reply:
Yes thank you! :)